test/acceptance/features/customEnvVar.feature

```diff
@@ -0,0 +1,128 @@
+                host: example.common
+                tags:
+                    - "centos7-12.3"
+                    - 123
+            """
+    * Generic test application "foo" is running
+    When Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: custom-env-var-from-sequence
+            spec:
+                application:
+                    name: foo
+                    group: apps
+                    version: v1
+                    resource: deployments
+                services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-with-tag-sequence
+                    id: backend
+                customEnvVar:
+                   - name: TAGS
+                     value: '{{ .backend.spec.tags }}'
+            """
+    Then The application env var "TAGS" has value "[centos7-12.3 123]"
+
+
+  Scenario: Map from service resource is injected into application using custom env variables without specifying annotations
+    Given OLM Operator "backend" is running
+    * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-with-user-labels-map
+            spec:
+                host: example.common
+                userLabels:
+                    archive: "false"
+                    environment: "demo"
+            """
+    * Generic test application "foo" is running
+    When Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: custom-env-var-from-map
+            spec:
+                application:
+                    name: foo
+                    group: apps
+                    version: v1
+                    resource: deployments
+                services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-with-user-labels-map
+                    id: backend
+                customEnvVar:
+                   - name: USER_LABELS
+                     value: '{{ .backend.spec.userLabels }}'
+            """
+    Then The application env var "USER_LABELS" has value "map[archive:false environment:demo]"
+
+
+  Scenario: Scalar from service resource is injected into application using custom env variables without specifying annotations
+    Given OLM Operator "backend" is running
+    * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-with-user-labels-archive
+            spec:
+                host: example.common
+                userLabels:
```

Comment:
We have the same example for the CR mentioned in the `Map` case as well

Reply:
The same example, but this time we refer just particular value inside that map.